ember-flight-icons/tests/dummy/app/templates/engineering.hbs

```diff
@@ -9,6 +9,25 @@
   <p>🚨 Note: All of the flight-icon npm packages are currently in beta and any production usage should be discussed with the Design Systems team first.</p>
 </div>
 
+<div class="mt-8 prose prose-lg text-gray-800 pb-6">
+  <h2 class="ds-h2" id="accessibility"><a href="#accessibility" class="ds-a" rel="external">&sect;</a> Accessibility</h2>
+  <p>
+  Accessibility (a11y) support for SVGs is inconsistent across browsers and assistive technology. Currently, best practice is to set the <kbd class="ds-kbd">aria-hidden</kbd> attribute to false on the SVG itself.
+  This means that the icon (both the singular icon and the icon component) will need to be used <em class="ds-em">in context</em>.
+  The icons themselves are for presentation purposes only and should never be used on their own.
+  </p>
+  <p>However: As a <em>temporary</em> bridge while we work to provide the accessible components in the design system, we have provided the ability to add a title element to the Ember component by defining a value for the <kbd class="ds-kbd">@title</kbd> property. 
```

Comment:
This note seems misleading to me as (I think) we've shown several uses of icons in our products where they are used to replace text content but are not interactive.

Reply:
Thank you for your feedback. We are encouraging folks to work with their designers to move on from this pattern. 👍